ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -546,7 +551,7 @@ tests:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-      TEST_ADDITIONAL: Hypershift
+      TEST_ADDITIONAL: Hypershift&;Network_Observability
```

Comment:
@kuiwang02 - is this correct way to enable for more than one?

Reply:
@memodi oh, no. I am sorry I was not aware of this.
it should be `TEST_ADDITIONAL: Hypershift|Network_Observability`

Reply:
Fixed it, should be all set now.